docs/conf.py

```diff
@@ -22,7 +22,7 @@
 
 project = "pyscal"
 author = u"Håvard Berland"
```

Comment:
```suggestion
author = "Håvard Berland"
```
Strings are by default unicode on Python 3.